airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -0,0 +1,2668 @@
+#
```

Comment:
I ran this file and the old `airflow/auth/managers/fab/security_manager/override.py` though a diff, for easier review:  https://editor.mergely.com/kNuiVsA7

Changes are mainly to import paths and comments.